python/paddle/v2/fluid/tests/unittests/test_pool2d_op.py

```diff
@@ -79,8 +80,10 @@ def setUp(self):
             'ksize': self.ksize,
             'pooling_type': self.pool_type,
             'global_pooling': self.global_pool,
+            'data_format':
+            'AnyLayout',  # TODO(dzhwinter) : should be fix latter
             'use_cudnn': self.use_cudnn,
-            'data_format': 'AnyLayout'  # TODO(dzhwinter) : should be fix latter
```

Comment:
From my perspective, it's not appropriate we remove this TODO.
I suggest we remain it and let the owner to fix it, since we may miss some background information.

Reply:
I did not remove this line, it's been moved up.